### PR TITLE
feat: fixed all images in experience cloud site

### PR DESCRIPTION
**Pull request by pozil**
Images speak better than words :)

BTW I also fixed the sample data and our S3 content to use the new experience images.

Screenshot 2024-11-06 at 14 23 34

### CODE REVIEW

File: cc-service-app/main/default/networks/coral cloud.network-meta.xml [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
+    <enableDirectMessages>false</enableDirectMessages>
     <enableExpFriendlyUrlsAsDefault>false</enableExpFriendlyUrlsAsDefault>
     <enableExperienceBundleBasedSnaOverrideEnabled
     >true</enableExperienceBundleBasedSnaOverrideEnabled>
-    <enableGuestChatter>false</enableGuestChatter>
+    <enableGuestChatter>true</enableGuestChatter>
     <enableGuestFileAccess>true</enableGuestFileAccess>
     <enableGuestMemberVisibility>false</enableGuestMemberVisibility>
-    <enableImageOptimizationCDN>true</enableImageOptimizationCDN>
+    <enableImageOptimizationCDN>false</enableImageOptimizationCDN>
```
Comment (muenzpraeger) [inline cc-service-app/main/default/networks/coral cloud.network-meta.xml:23]:
```suggestion
    <enableImageOptimizationCDN>true</enableImageOptimizationCDN>
```

File: cc-service-app/main/default/digitalExperiences/site/coral_cloud1/sfdc_cms__view/home/content.json [hunk 2]
```diff
@@ -72,8 +73,8 @@
                                             "attributes": {
                                                 "deploymentName": "Agent_Web_Deployment",
                                                 "hideChatButtonOnLoad": "Default",
-                                                "scrtUrl": "https://%%SF_CC_PLACEHOLDER_DOMAIN%%.develop.my.salesforce-scrt.com",
-                                                "siteEndpoint": "https://%%SF_CC_PLACEHOLDER_DOMAIN%%.develop.my.site.com/ESWAgentWebDeployment1729699014420"
+                                                "scrtUrl": "https://coralcloudresorts-3a-dev-ed.develop.my.salesforce-scrt.com",
```
Comment (muenzpraeger) [inline cc-service-app/main/default/digitalExperiences/site/coral_cloud1/sfdc_cms__view/home/content.json:76]:
```suggestion
                                                "scrtUrl": "https://%%SF_CC_PLACEHOLDER_DOMAIN%%.develop.my.salesforce-scrt.com",
```

File: cc-service-app/main/default/digitalExperiences/site/coral_cloud1/sfdc_cms__view/home/content.json [hunk 3]
```diff
@@ -72,8 +73,8 @@
...
+                                                "siteEndpoint": "https://coralcloudresorts-3a-dev-ed.develop.my.site.com/ESWAgentWebDeployment1729699014420"
```
Comment (muenzpraeger) [inline cc-service-app/main/default/digitalExperiences/site/coral_cloud1/sfdc_cms__view/home/content.json:77]:
```suggestion
                                                "siteEndpoint": "https://%%SF_CC_PLACEHOLDER_DOMAIN%%.develop.my.site.com/ESWAgentWebDeployment1729699014420"
```

File: cc-service-app/main/default/networks/ESW_Agent_Web_Deployment_1729699014420.network-meta.xml [hunk 4]
```diff
@@ -21,7 +21,7 @@
     <enableGuestChatter>false</enableGuestChatter>
     <enableGuestFileAccess>false</enableGuestFileAccess>
     <enableGuestMemberVisibility>false</enableGuestMemberVisibility>
-    <enableImageOptimizationCDN>true</enableImageOptimizationCDN>
+    <enableImageOptimizationCDN>false</enableImageOptimizationCDN>
```
Comment (muenzpraeger) [inline cc-service-app/main/default/networks/ESW_Agent_Web_Deployment_1729699014420.network-meta.xml:24]:
```suggestion
    <enableImageOptimizationCDN>true</enableImageOptimizationCDN>
```

**Review by muenzpraeger — CHANGES_REQUESTED** [review]
LGTM

**Comment by pozil** [comment]
Sorry about messing up the string replacement variables. I had taken care not to overwrite them several times and I didn't pay attention on the last run.
This is bound to happen again whether it's me or someone else :(

**Review by muenzpraeger — APPROVED** [review]

**Pull request merged**